napari/layers/base/base.py

```diff
@@ -130,17 +131,16 @@ def __init__(
         self.scale_factor = 1
 
         self.dims = Dims(ndim)
+
         if scale is None:
-            self._scale = [1] * ndim
-        else:  # covers list and array-like inputs
-            self._scale = list(scale)
+            scale = [1] * ndim
```

Comment:
Will this still work in cases where a (not None) scale argument is passed in? I don't see a line for that setting the `self._scale` attribute.